examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/fragment/_cq_template.xml

```diff
@@ -2,4 +2,5 @@
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="nt:unstructured"
     jcr:title="Fragment"
+    fragmentPath=""
```

Comment:
Why do we need an empty key here ?

Reply:
I would prefer fd:viewType rather than fragmentPath, since this is not intuitive.